Dockerfile.production

```diff
@@ -81,7 +81,7 @@ RUN mv /opt/formsg/dist/backend/shared /opt/formsg/
 # https://www.npmjs.com/package/puppeteer-core?activeTab=versions for corresponding versions
 
 RUN apk add --no-cache \
-# Compatible chromium versions can be found here https://pkgs.alpinelinux.org/packages?name=chromium&branch=v3.18&repo=&arch=&maintainer=
+    # Compatible chromium versions can be found here https://pkgs.alpinelinux.org/packages?name=chromium&branch=v3.18&repo=&arch=&maintainer=
     chromium=119.0.6045.159-r0 \
```

Comment:
Should we update chromium for our dockerfiles in tandem with the alpine upgrades? and update this comment as well. The link [here](https://pkgs.alpinelinux.org/packages?name=chromium&branch=v3.19&repo=&arch=&maintainer=) seems to suggest that the supported version of Chromium is `121.0.6167.85-r0`

Reply:
Great catch! Updated chromium version